src/Angle.re

```diff
@@ -22,9 +22,9 @@ let to_degrees = (angle: t) => {
 };
 
 let cos = (angle: t) => {
-    Pervasives.cos(to_radians(angle));
+    cos(to_radians(angle));
```

Comment:
Pervasives is not needed because we haven't rec flag
And it will force to add stdlib-shim

Reply:
Cool, thank you for the explanations! 👍